lib/tasks/scripts/telemedicine_reports.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module TelemedicineReports
+  class << self
+    def parse_mixpanel(mixpanel_csv_path)
+      mixpanel_csv = File.read(mixpanel_csv_path)
+      mixpanel = CSV.parse(mixpanel_csv, headers: false)
```

Comment:
Instead of `headers: false` and dropping 1 below, can we use `headers: true`, and use headers names to access data instead of `row[1]`, etc?

Reply:
The last two headers are dynamic unfortunately, can't predict what they're going to be.
![Screenshot 2020-06-25 at 4 41 20 PM](https://user-images.githubusercontent.com/48791115/85709482-ed766b80-b702-11ea-824a-153cc05d6e7b.png)

Reply:
Fuuuuun, coolbeans 👍